.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: node_js
 node_js:
-  - lts/*
+  # we recommend testing addons with the same minimum supported node version as Ember CLI
```

Comment:
My understanding is that Travis builds are no longer free. Do we want to migrate this over to GitHub Actions? 

Also if not, I've never touched Travis before so if someone could verify that this YAML file is still valid that would be great.